fastf1/core.py

```diff
@@ -894,37 +894,38 @@ def calculate_driver_ahead(self):
         for drv in self.session.drivers:
             # find correct first relevant lap; very important for correct zero point in distance
             drv_laps = self.session.laps[self.session.laps['DriverNumber'] == drv]
-            drv_laps_before = drv_laps[(drv_laps['LapStartTime'] <= t_start)]
-            if not drv_laps_before.empty:
-                lap_n_before = drv_laps_before['LapNumber'].iloc[-1]
-                if lap_n_before < first_lap_number:
-                    # driver is behind on track an therefore will cross the finish line AFTER self
-                    # therefore above check for LapStartTime <= t_start is wrong
-                    # the first relevant lap is the first lap with LapStartTime > t_start which is lap_n_before += 1
-                    lap_n_before += 1
-            else:
-                lap_n_before = min(drv_laps['LapNumber'])
+            if not drv_laps.empty: # Only include drivers who participated in this session
```

Comment:
I'd prefer to have 
```
if drv_laps.empty:
    continue
```
The line lengths are getting a bit out of hand if you indent all the following code.